src/Eris/Generator/FrequencyGenerator.php

```diff
@@ -16,6 +16,7 @@ function frequency(/*$frequencyAndGenerator, $frequencyAndGenerator, ...*/)
 class FrequencyGenerator implements Generator
 {
     private $generators;
+    private $frequencies;
```

Comment:
@giorgiosironi 

Problem is, I have no clue right now what should happen, hehe! Can you help?